scripts/process_GMAP_alignments_gff3_chimeras_ok.pl

```diff
@@ -88,7 +88,7 @@
 
 	my $cmd = "gmap -D $genomeBaseDir -d $genomeDir $transcriptDB -f $format -n $num_gmap_top_hits -x 50 -t $CPU -B 5 ";
 	if ($max_intron) {
-		$cmd .= " --intronlength=$max_intron ";
+		 $cmd .= "--max-intronlength-middle=".$max_intron." --max-intronlength-ends=".$max_intron;
```

Comment:
looks like this was captured as:

if ($max_intron) {
                my $gmapv=`gmap --version 2>&1 | head -1`;
                chomp($gmapv);
                $gmapv=~/GMAP\s+version\s+(\d{4})-(\d{2})-.*/;
                if( ($1 == 2016 && $2 >= 5) || $1 > 2016){
                        $cmd .= " --max-intronlength-middle=$max_intron --max-intronlength-ends=$max_intron ";
                }else{
                        $cmd .= " --intronlength=$max_intron ";
                }